hawkbit-autoconfigure/src/main/java/org/eclipse/hawkbit/autoconfigure/security/InMemoryUserManagementAutoConfiguration.java

```diff
@@ -57,17 +65,62 @@ public void configure(final AuthenticationManagerBuilder auth) throws Exception
     @Bean
     @ConditionalOnMissingBean
     UserDetailsService userDetailsService() {
-        final InMemoryUserDetailsManager inMemoryUserDetailsManager = new InMemoryUserPrincipalDetailsManager();
-        inMemoryUserDetailsManager.setAuthenticationManager(null);
-        final SecurityProperties.User user = securityProperties.getUser();
-        final UserBuilder userBuilder = User.builder().username(user.getName()).password(user.getPassword())
-                .authorities(PermissionUtils.createAllAuthorityList());
-        final List<String> roles = user.getRoles();
-        if (!roles.isEmpty()) {
-            userBuilder.roles(roles.toArray(new String[roles.size()]));
+        final String defaultTenant = "DEFAULT";
```

Comment:
May I ask you to extract tenant as constant and reuse it within `TenantDaoAuthenticationProvider#createSuccessAuthentication` (PR [line 142](https://github.com/eclipse/hawkbit/blob/master/hawkbit-autoconfigure/src/main/java/org/eclipse/hawkbit/autoconfigure/security/InMemoryUserManagementAutoConfiguration.java#L89)) as well?